lib/vagrant-openshift/action/run_sti_tests.rb

```diff
@@ -52,7 +52,7 @@ def call(env)
           _,_,env[:test_exit_code] = sudo(env[:machine], %{
 set -e
 pushd #{Constants.build_dir}/source-to-image >/dev/null
-export PATH=/data/src/github.com/openshift/source-to-image/_output/local/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64:$PATH
+export PATH=/data/src/github.com/openshift/source-to-image/_output/local/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64:/data/src/github.com/openshift/source-to-image/_output/local/bin/windows/amd64:$PATH
```

Comment:
Agreed - will revert this change, update s2i and add a comment indicating that the entire PATH line can be removed once the s2i change merges.